### PR TITLE
Final refactoring of Gytis

### PR DESCRIPTION
**Pull request by Stokalas**
Refactored a lot of files, but all changes should be small.

### CODE REVIEW

File: SmartSaver/DataManagement/DataTotals.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Linq;
+using ePiggy.Utilities;
+
+namespace ePiggy.DataManagement
+{
+    public class DataTotals
```
Comment (ciliakas) [inline SmartSaver/DataManagement/DataTotals.cs:7]: Good idea to move these methods, maybe the name of the class should be DataTotaler or something like that, to reflect that this is not something like a structure to hold information.

**Review by ciliakas — APPROVED** [review]
Just a well done refactoring overall, style is according to standard, everything functions just as it should.

**Review by DeividasLT5 — APPROVED** [review]
Nice refactorization, couldn't find issues with it. If's fit our standard, there's added "var"s I like, and even a new constant. Good job

File: SmartSaver/DataManagement/DataTotals.cs (hunk 1 above)
Comment (Stokalas) [reply SmartSaver/DataManagement/DataTotals.cs]: This suggestion makes sense, I've implemented changes according to it.

**Pull request merged**